tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigPropertiesTest.java

```diff
@@ -256,12 +251,11 @@ public void testConfigPropertiesDefaultOnBeanPL() {
         Config config = ConfigProvider.getConfig();
         BeanFour myBeanFour = config.getConfigProperties(BeanFour.class);
         Assert.assertEquals("mycloud.org", myBeanFour.getHost());
-        Assert.assertEquals("9080", myBeanFour.port);
+        Assert.assertEquals(9080, myBeanFour.port);
         Assert.assertFalse(myBeanFour.location.isPresent());
     }
 
     @ConfigProperties(prefix="customer.")
-    @Dependent
```

Comment:
I thought the Bean defining annotation are required by Quarkus. In FT, we had to explicitly add the scope annotation even though there is a beans.xml. Is this not the case any more?

Reply:
I guess it depends on how you implement it, but we don't need it.

We just scan for annotated beans with `@ConfigProperties` and create a configurator bean with `@Dependent` scope directly. We don't honor any scope defined in the class itself.

Reply:
What if the class with `@ConfigProperties` annotation also declares a scope, say `@RequestScoped`, the end user would like the lifecyle to be with `@RequestScoped` not `@Dependent` scoped. I am talking about the bean class with `@ConfigProperties`.

Reply:
I think we should not support it for now. It will become confusing with `@ConfigProperty` which is only Dependent scope.

Reply:
`@ConfigProperty` has no scope associated with as it is just a qualifier not a stereotype.

Reply:
same thing for `@ConfigProperties`.

Reply:
Ok, it's fine.